rxbindings/src/main/java/com/amplifyframework/rx/RxDataStoreBinding.java

```diff
@@ -72,6 +72,13 @@
             dataStore.delete(item, predicate, onResult, onError));
     }
 
+    @NonNull
+    @Override
+    public <T extends Model> Completable delete(@NonNull Class<T> itemClass, @NonNull QueryPredicate predicate) {
+        return toCompletable((onResult, onError) ->
+                dataStore.delete(itemClass, predicate, () -> {}, onError));
```

Comment:
For the `() -> {}`, I think we have a `NoOpConsumer.create()` or something of that sort, which is a minor readability enhancement. Makes it look more deliberate.

Reply:
this overload of `delete` takes in `Action` rather than a `Consumer`. There doesn't appear to be a no-op Action available outside of `EmptyAction` which resides in `testutils` package.

Reply:
added `NoOpAction` to address this.